Mathlib/CategoryTheory/EpiMono.lean

```diff
@@ -38,7 +38,6 @@ such that `f ≫ retraction f = 𝟙 X`.
 Every split monomorphism is a monomorphism.
 -/
 /- Porting note(#5171): removed @[nolint has_nonempty_instance] -/
-/- Porting note: `@[ext]` used to accept lemmas like this. Now we add an aesop rule -/
 @[ext, aesop apply safe (rule_sets := [CategoryTheory])]
 structure SplitMono {X Y : C} (f : X ⟶ Y) where
```

Comment:
Should the `aesop` tag be removed accordingly? What does it even do, when applied to a structure?

Reply:
So I went as far as checking the `aesop` source code and as far as I can tell, no idea what it's supposed to mean 🤷. Looks like removing the attribute doesn't break anything either.

Reply:
Thanks for the digging! @JLimperg, maybe there should be a check in aesop that one is not tagging a structure?

Reply:
The rule takes the `structure` and register it as a rule with conclusion `Type _`, naturally. :slightly_smiling_face: But yes, Aesop should warn about this. leanprover-community/aesop#172